Cargo.toml

```diff
@@ -21,5 +21,17 @@ memmap = "0.7.0"
 num-traits = "0.2"
 thiserror = "1.0"
 
+[features]
+default = []
+
+# Enable this feature to use it with the
+# synapticon branch `release/v1.5.2-sncn-11`
+# at https://github.com/synapticon/Etherlab_EtherCAT_Master
+sncn = []
+
+[patch.crates-io.ethercat-types]
```

Comment:
this is just temporary I assume?

Reply:
sure. I modified one struct of `ethercat-types` that leads to a breaking change. For now I'd like to play around with different use cases before releasing `v0.3.0` of `ethercat-types`.
The overall roadmap currently is documented in issue #12.